.tours/dotnet6/http-refit/.tours/01-defining-a-third-party-rest-api.tour

```diff
@@ -0,0 +1,81 @@
+{
+  "$schema": "https://aka.ms/codetour-schema",
+  "title": "1-defining-a-third-party-rest-api",
+  "steps": [
+    {
+      "title": "Goal",
+      "description": "The goal of this lesson is to show how to call third-party REST APIs while using best practices.\r\n\r\nCalling REST APIs usually involves the well-known [HttpClient](https://docs.microsoft.com/fr-fr/dotnet/api/system.net.http.httpclient?view=net-6.0) .NET class. However, using this class effectively has always been a challenge as it is designed to be instantiated once and reused troughout the life of your application.\r\n\r\nStarting from .NET Core 2.1, the BCL [introduced a set of changes](https://docs.microsoft.com/en-us/dotnet/architecture/microservices/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests) designed to make `HttpClient` easier to use in a correct and efficient manner."
+    },
+    {
+      "file": "../../../src/dotnet6/http-refit/AzFuncUni.Http/AzFuncUni.Http.csproj",
```

Comment:
Can you add a title field for all the steps? That looks a bit better than the default path. I usually do: *FileName.ext - Keyword*. For example: *IHttpBinOrgApi.cs - GET*